src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx

```diff
@@ -50,7 +50,7 @@ export function getQuery(view, location) {
   data.field = [...new Set(fields)];
   data.groupby = groupby;
   data.sort = view.data.sort;
-
+  data.per_page = 50;
```

Comment:
Do we explicitly set per_page to 50 on frontend for other resources?

Reply:
We do, and we seem to use 50 most commonly (with the notable exception of the issue list)

Reply:
Would this be better as an application level constant?

Reply:
I think so, will add it here